### DISCUSSION TITLE
License of code snippets in the wiki

### DISCUSSION THREAD

**Discussion by ssalonen** [cross-rs/cross#1297] [General]
Can you please clarify the license of code snippets in the wiki, specifically https://github.com/cross-rs/cross/wiki/Recipes

**Comment by Emilgardis** [comment]
All content on the wiki fall under https://github.com/cross-rs/cross#license.

However (note that I am not a lawyer), I think the snippets are to general to even be considered to be part of some type of licensing, especially since it's configuration.

**Reply by Emilgardis** [reply]
Note, if you think it is needed, we could surely add one if @Alexhuszagh is ok with licensing it separately, it's only them and me that has written stuff there.

**Reply by ssalonen** [reply]
This is what I was was assuming although it is not explicit. Thanks!
